### PR TITLE
Known kssonet issues added.

### PR DESCRIPTION
**Pull request by Akado2009**
Fixes: https://github.com/kubeflow/website/issues/213

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/216)

### CODE REVIEW

File: content/docs/guides/troubleshooting.md [hunk 1]
```diff
@@ -172,3 +172,12 @@ ks version
...
 
 If your ksonnet version is lower than what is specified in the [requirements](/docs/guides/requirements), please upgrade it and follow the [guide](/docs/guides/components/ksonnet) to recreate the app.
+
+## Ksonnet on Windows
+There are some known issues with ksonnet and Windows. You might consider 
+alternative solutions.
+
+ * construct base object: Failed to filter components; the following 
+ components don't exist:  ['kubeflow-core']  [#481](https://github.com/kubeflow/kubeflow/issues/481)
+ * "ks apply" fails to correctly process paths in Windows shell [#382](https://github.com/ksonnet/ksonnet/issues/382)
```
Comment (sarahmaddox) [inline content/docs/guides/troubleshooting.md]: Let's change this:
[#382]
to this:
[ksonnet #382]

My reasoning is that readers may think the issue number refers to a Kubeflow issue, following the pattern of the previous bullet point. If we add the word "ksonnet" that makes it immediately clear that the issue is in another repo.

File: content/docs/guides/troubleshooting.md [hunk 2]
```diff
@@ -172,3 +172,12 @@ ks version
 ```
 
 If your ksonnet version is lower than what is specified in the [requirements](/docs/guides/requirements), please upgrade it and follow the [guide](/docs/guides/components/ksonnet) to recreate the app.
+
+## Ksonnet on Windows
```
Comment (sarahmaddox) [inline content/docs/guides/troubleshooting.md]: Change "Ksonnet" to "ksonnet".
Hint from a tech writer: Even though the word is at the beginning of the sentence, it's still good practice to follow the product's branding choice.

**Comment by Akado2009** [comment]
@sarahmaddox  Thank you for the commentary! Everything is fixed now and commits are squashed.

File: content/docs/guides/troubleshooting.md [hunk 3]
```diff
@@ -172,3 +172,12 @@ ks version
 ```
 
 If your ksonnet version is lower than what is specified in the [requirements](/docs/guides/requirements), please upgrade it and follow the [guide](/docs/guides/components/ksonnet) to recreate the app.
+
+## ksonnet on Windows
+There are some known issues with ksonnet and Windows. You might consider 
+alternative solutions.
+
+ * construct base object: Failed to filter components; the following 
+ components don't exist:  ['kubeflow-core'] [kubeflow #481](https://github.com/kubeflow/kubeflow/issues/481)
```
Comment (sarahmaddox) [inline content/docs/guides/troubleshooting.md]: This link is [rendering oddly now](https://deploy-preview-216--competent-brattain-de2d6d.netlify.com/docs/guides/troubleshooting/) - it looks like this on the page:
[‘kubeflow-core’] #481

Suggested markup and reduced text:

```
 * construct base object: Failed to filter components ([kubeflow #481](https://github.com/kubeflow/kubeflow/issues/481))
```

File: content/docs/guides/troubleshooting.md [hunk 4]
```diff
@@ -172,3 +172,12 @@ ks version
...
+ * "ks apply" fails to correctly process paths in Windows shell [[ksonnet #382]](https://github.com/ksonnet/ksonnet/issues/382)
```
Comment (sarahmaddox) [inline content/docs/guides/troubleshooting.md]: We don't need the double square brackets around the link text. Please would you try the following markup and see how it renders:

```
 * `ks apply` fails to correctly process paths in Windows shell ([ksonnet #382](https://github.com/ksonnet/ksonnet/issues/382))
```

**Comment by Akado2009** [comment]
@sarahmaddox My bad. Everything's fixed now.

**Comment by sarahmaddox** [comment]
Thanks!
/lgtm

**Pull request merged**